cetloxblock/CetInsights/CetInsightsPage.html

```diff
@@ -0,0 +1,83 @@
+﻿<!--Add the following HTML code to CET Insights page's RAW html-->
+<script src="https://login.cet.ac.il/Scripts/login.js"></script>
+<script src="//ebag.dev.cet.ac.il/CetSso.js"></script>
+<script>
+  var CET_DOMAIN = ".lab.cet.ac.il";
+  if (window.location.host.indexOf("localhost") >= 0 || window.location.host.indexOf("127.0.0.1") >= 0) {
+    CET_DOMAIN = ".dev.cet.ac.il"
+  }
+  var PRODUCT_PLAYER = "//productplayer" + CET_DOMAIN + "/";
+  var PRODUCT_PLAYER_SERVICES = "//productplayerservices" + CET_DOMAIN + "/";
+
+  function selectView() {
+    CetSso.getsession(function (user) {
+      if (user) {
+        if (user.userRole == "student") {
+          studentView();
+        }
+        else {
+          teacherView();
+        }
+      }
+      else {
+        // IDM SSO
+        CET.Authentication.LogMeIn('IDM');
+      }
+    });
+  }
+
+  this.getCourseEdxName = function () {
+    var match = /-v\d+:([^\/\+]+\+[^\/\+]+\+[^\/\+]+)[\/\+]/gi.exec(location.pathname);
+    if (match.length) {
+      return match[1];
+    }
+  }
+
+  function getCourseCetName(edxName) {
+    edxName = edxName || this.getCourseEdxName();
+    var url = PRODUCT_PLAYER + "edxCourses.json";
+    return $.Deferred(function (dfd) {
+      $.get(url).done(function (response) {
+        var courseObj = response[edxName];
+        if (courseObj) {
+          dfd.resolve(courseObj.path);
+        }
+        else {
+          dfd.reject("Cet course is not defined");
+        }
+      }).fail(function (response) {
+        dfd.reject("Failed to retrieve Cet course name. Error: " + response);
+      });
+    });
+  }
+
+  function setLink(courseCetName) {
+    var link = document.getElementById('cet-insights-link');
+    link.href = "//productplayer" + CET_DOMAIN + "/" + courseCetName;

```

Comment:
Similar recommendation as above -- we should be explicit about the protocol.

Also we should use `PRODUCT_PLAYER + courseCetName` here I think since we defined it above.